example/jitter/memstruct.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python
+"""This script is just a short example of common usages for miasm2.analysis.mem.
```

Comment:
It seems that *memstruct* are more related to `miasm2.core`, as `DiGraph` than analysis module, such as `miasm2.analysis.depgraph`.